css/properties.json

```diff
@@ -4753,40 +4763,40 @@
       "CSS Motion"
     ],
     "initial": "<code>0</code>",
-    "appliesto": "allElementsSVGContainerElements",
-    "computed": "asSpecified",
-    "order": "uniqueOrder",
+    "appliesto": "transformableElements",
+    "computed": "forLengthAbsoluteValueOtherwisePercentage",
+    "order": "perGrammar",
     "status": "experimental"
   },
   "offset-path": {
-    "syntax": "&lt;url&gt; | [ &lt;basic-shape&gt; | &lt;path()&gt; ] || &lt;geometry-box&gt; | none",
+    "syntax": "none | ray( [ &lt;angle&gt; && &lt;size&gt;? && contain? ] ) | &lt;path()&gt; | &lt;url&gt; | [ &lt;basic-shape&gt; || &lt;geometry-box&gt; ]",
     "media": "visual",
     "inherited": false,
-    "animationType": "discrete",
+    "animationType": "AsAngleBasicshapeOrPath",
```

Comment:
Nit: We're using lowerCamelCase for the keywords, so that should be `asAngleBasicShapeOrPath`.

Sebastian